src/tests/Microsoft.PowerFx.Core.Tests/TexlTests.cs

```diff
@@ -3620,6 +3620,54 @@ public void TestAttachmentDottedNameNodeBinding(string script, string expectedSc
             symbol.AddEntity(new TestDataSource("DS", schema));
 
             TestSimpleBindingSuccess(script, expectedType, symbol);
+        }
+
+        [Theory]
+        [InlineData("Search(T, \"a\", \"Name\")", "*[Name:s]")]

```

Comment:
>\"Name\" [](http://example.com/codeflow?start=38&length=8)

We should have some tests with column names as identifiers as well.

Reply:
Added.